Cargo.toml

```diff
@@ -26,13 +26,17 @@ bitflags = { version = "1.2", default-features = false }
 clap = { version = "3.1", features = ["env", "derive", "std"], default-features = false }
 colorful = { version = "0.2", default-features = false }
 dirs = { version = "4.0", default-features = false }
+drawbridge-client = { git = "https://github.com/profianinc/drawbridge", rev = "d8e0befbd8da6b9f357c396273996e6141f7569f", default-features = false }
```

Comment:
We should release Drawbridge crates, I'll take care of that

Reply:
Sure, although note that they all need explicit licenses in their Cargo.tomls.